docs(success-stories): Add Biden for President quote

prophen opened the pull request:
I added the quote from Matt Hodges, Engineering Director, Biden For President to the testimonials section in `/success-stories`

prophen committed:
docs(success-stories): Add Biden for President quote

prophen commented:
@brian-armory please review, thank you

aimeeu reviewed (changes requested):
@prophen Nikema - thanks for the update. Couple of concerns - since there's no logo, that space appears as a broken image.  Also, there's nothing in the "Click here to learn more" section.  It looks like these success stories involved interviewing the user? Not really sure how to proceed with this.

https://s.armory.io/WnuBoq05

prophen committed:
Add image to Biden for President quote and move to Case Studies

I moved the quote to the Case Studies section and provided an image so that it doesn't break the page design.

prophen commented:
> @prophen Nikema - thanks for the update. Couple of concerns - since there's no logo, that space appears as a broken image. Also, there's nothing in the "Click here to learn more" section. It looks like these success stories involved interviewing the user? Not really sure how to proceed with this.
> 
> https://s.armory.io/WnuBoq05

Thanks, @aimeeu. I addressed your concerns with the latest commit. The quote is now in the case studies section along with an image so it doesn't break the design 
![image](https://user-images.githubusercontent.com/3941856/111725053-8b770c80-8823-11eb-8f14-8aba43b93514.png)

prophen committed:
Merge branch 'master' into prophen-add-testimonial

aimeeu commented on _data/success-stories.yml:
```diff
@@ -25,6 +25,11 @@ cases:
     text: Thanks to Spinnaker, developers can focus on developing business logic rather than becoming experts on each cloud platform. Teams can track the lifecycle of a deployment using several notification mechanisms including email, Slack, and SMS, allowing them to coordinate handoffs between developer and QA teams.
 
+  - name: Biden for President
+    link: https://spinnaker.io/guides/
```
Wondering why this link is to the Spinnaker docs? In other Case Studies, the "Learn" link goes to a whitepaper or external site with an actual case study. Is it possible to link to wherever that quote appeared?  twitter, news site, ??

prophen committed:
Add Featured Quotes section to /success-stories

I added a section for quotes so that there would be a logical place for the Biden for President quote.

prophen committed:
Merge branch 'master' of github.com:spinnaker/spinnaker.github.io into prophen-add-testimonial

Update working branch with merged code

prophen committed:
Merge branch 'prophen-add-testimonial' of github.com:prophen/spinnaker.github.io into prophen-add-testimonial
Update working branch

prophen commented:
![image](https://user-images.githubusercontent.com/3941856/112063517-c24c6b80-8b1e-11eb-9df5-9e8e40394b38.png) I created a new section for the quote so that it isn't out of place with the testimonials or case studies. What do you think, @aimeeu ?
